packages/cli/src/commands/rolling-release/command.ts

```diff
@@ -1,82 +1,178 @@
+import { packageName } from '../../util/pkg-name';
+
 export const rollingReleaseCommand = {
   name: 'rolling-release',
-  aliases: ['rr'],
+  aliases: ['rr', 'release'],
   description: "Manage your project's rolling release.",
   arguments: [],
-  options: [
+  subcommands: [
     {
-      name: 'name',
-      shorthand: 'n',
-      type: String,
-      deprecated: true,
-      description: 'Provide a Vercel Project name',
+      name: 'configure',
+      description: 'Configure rolling release settings for a project',
+      aliases: [],
+      arguments: [],
+      examples: [
+        {
+          name: 'Configure a new rolling release with an intial stage of 10% that lasts for 5 minutes before automatically advancing to 100%',
+          value: `${packageName} rolling-release configure --cfg='{"enabled":true, "advancementType":"automatic", "stages":[{"targetPercentage":10,"duration":5},{"targetPercentage":100}]}'`,
+        },
+      ],
+      options: [
+        {
+          name: 'cfg',
+          shorthand: null,
+          deprecated: false,
+          type: String,
+          description: "The project's rolling release configuration",
+        },
+      ],
     },
     {
-      name: 'action',
-      shorthand: null,
-      type: String,
-      deprecated: false,
-      description: "Action to perfom on a project's rolling release.",
-      enum: ['configure', 'start', 'approve', 'abort', 'complete', 'fetch'],
+      name: 'start',
+      description: 'Start a rolling release',
+      aliases: [],
+      arguments: [],
+      examples: [
+        {
+          name: 'Start a rolling release',
+          value: `${packageName} rolling-release start --deployId=dpl_123`,
+        },
+      ],
+      options: [
+        {
+          name: 'deployId',
```

Comment:
Can we change this from `deployId` to `dpl`? That way you could do

```sh
vc release start --dpl=dpl_123
```

Or even shorter

```sh
vc release start --dpl=123
```